src/app/content/highlights/components/SummaryPopup/constants.ts

```diff
@@ -2,4 +2,5 @@ export const mobilePadding = {
     leftRight: 1.6,
     topBottom: 1.4,
 };
-export const mobileMargin = 0.8;
+export const mobileMarginSides = 0.8;
+export const mobileMarginTopBottom = 2;
```

Comment:
the variables in this file could be formatted more consistently... i like `mobileMarginSides` and `mobileMarginTopBottom` (or `mobileMarginHorizontal` and `mobileMarginVertical` would also be cool) but `mobilePadding` is a subobject with `leftRight` and `topBottom` properties.